qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -144,6 +143,12 @@ def widgets(self):
             w.append(self.widget(i))
         return w
 
+    @config.change_filter('tabs', 'tabbar-size')
+    def set_icon_size(self):
+        """Take the (fav)icon size from the config."""
+        tabbar_size = config.get('tabs', 'tabbar-size')
+        self.setIconSize(QSize(tabbar_size, tabbar_size))
```

Comment:
With the change below, this should use the *actual* rather than the configured size - i.e. probably something like `self.tabBar().height()`.